Transportation/ServiceNetwork/Itinerary/doc/spec.md

```diff
@@ -0,0 +1,88 @@
+# Itinerary
+
+## Description
+
+An itinerary scheduled for a time period, it can be for just one day (by setting start and end dates) or fixed for the entire life. 
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `VehicleItinerary`. 
+
++ `name` : Name given to the itinerary
+    + Normative References: [https://schema.org/name](https://schema.org/name)
+    + Optional
+
++ `description` : Description about the itinerary. 
+    + Normative References: [https://schema.org/description](https://schema.org/description)
+    + Optional
+
++ `refRoute`: Route this itinerary belongs to.
+    + Attribute type: Reference to a [Route](../../Route/doc/spec.md)
+    + Mandatory
+
++ `vehicleType`: Describes the type of transportation used on a route
+    + Attribute type: [Number](https://github.com/schema.org/Number)
+    + Allowed values:
+        + `tram` : Any light rail or street level system within a metropolitan area
+        + `subway` : Any underground rail system within a metropolitan area
+        + `rail` : Used for intercity or long-distance travel
+        + `bus` : Used for short- and long-distance bus routes
+        + `ferry` : Used for short- and long-distance boat service
+        + `cablecar` : Used for street-level cable cars where the cable runs beneath the car
+        + `gondola` : Suspended cable car. Typically used for aerial cable cars where the car is suspended from the cable
+        + `funicular` : Any rail system designed for steep inclines
+        + `car` : Particular vehicle
+        + `van` : 
+        + `truck` : 
+        + `electriccar` :
+        - Any other value meaningfull for the scenario
+
+    + Mandatory
+
++ `startDate` : When the itinerary weeekdays start.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `endDate` : When the itinerary weekdays end.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `weekdays` : The weekdays that this trip refers to.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Allowed values (Informative):
+        + `monday'
+        + `tuesday`
+        + `wednesday`
+        + `thursday`
+        + `friday`
+        + `saturday`
+        + 'sunday`
+    + Optional
+
++ `scheduledPath` : Itinerary's scheduled path geometry represented by a GeoJSON LineString. 
+    + Normative References: [https://schema.org/description](https://schema.org/description)
+    + Optional
+
++ `performedPath` : Performed path of the vehicle, to be updated while vehicle is travelling. A geometry represented by a GeoJSON LineString. 
+    + Normative References: [https://schema.org/description](https://schema.org/description)
+    + Optional
+
```

Comment:
wouldn't it be better to decouple the modelling of the planned itinerary from the one of performed path? 
having a separate entity to model the actual itinerary, will avoid you to duplicate every time the static information of the itinerary. for example if you have a planned itinerary for a vehicle during a year, the plan want change, the only thing that will change is the actual itinerary run every day by the vehicle.

Reply:
+100